src/browser/index.ts

```diff
@@ -1,7 +1,7 @@
-// Include the root-level common exports so that the browser bundle
-// can export everything from a single module ("msw" import root).
-export * from '..'
+// Re-export the code MSW API (e.g. "rest", "RequestHandler") because
+// they are available from the "msw" (browser) root-level export.
+export * from '../core'
```

Comment:
shouldnt this be something like?
```suggestion
export * from '~/core'
```

Reply:
I thought so initially but for some strange reason, TS replaces the `~/core` import to an absolute path on the disk in the emitted files. I don't know why that happens but keeping a relative import here is fine since the depth of source and lib directories is the same for this module.